tests/drivers/pwm/pwm_loopback/src/test_pwm_loopback.c

```diff
@@ -125,6 +126,11 @@ ZTEST_USER(pwm_loopback, test_pulse_capture_inverted)
 
 ZTEST_USER(pwm_loopback, test_period_capture)
 {
+#if defined(CONFIG_SOC_SERIES_KE1XZ)
+	/* Skipping normal-polarity period capture in PWT tests isn't supported by ke1xz */
+	TC_PRINT("skip the period capture test in normal polarity\n");
```

Comment:
this should be handled by the driver, not by adding platform specific code to the test. You will notice that test_capture() will already skip if the driver returns -ENOTSUP when requesting this configuration.

Reply:
Thanks for reminding, I took this code to the driver, for not supported will return -ENOTSUP